rocky/rocky/templates/organizations/organization_member_list.html

```diff
@@ -56,6 +55,13 @@ <h2>{% translate 'Members' %}</h2>
                   {% endfor %}
                 {% endfor %}
               </td>
+              <td>
+                {% if member.verified %}
+                  {% include 'partials/single_action_checkbox_form.html' with input_checked=member.verified input_disabled=member.user.is_superuser action='unverify' key='member_id' value=member.id %}
+                {% else %}
+                  {% include 'partials/single_action_checkbox_form.html' with input_checked=member.verified input_disabled=member.user.is_superuser action='verify' key='member_id' value=member.id %}
+                {% endif %}
```

Comment:
At first, I was wondering why not pass the member as an object to the partial/ template, the template itself can decide what to do with it. But then I saw that this is used as a generic template/ component so this should be fine.

Although I hoped that we could achieve the same with less code, it's not easy to do so with the standard filters and tags Django templates provide.